db/seeds/custom_index_data/news.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+# After seeding the database, you should see these news items on the SERP and News facet:
+# /search?affiliate=test_affiliate&query=news
+# /search/news?affiliate=test_affiliate&channel=1&query=news
+
+affiliate = Affiliate.find_by(name: 'test_affiliate')
+
+rss_feed = affiliate.rss_feeds.new(
```

Comment:
My thinking in putting all this seed data in a single "news" file, as opposed to a file per model, was to make it easier for a developer to see all the data setup involved in news searches, without jumping through multiple seed files to do so. The idea is that we will eventually add a file for each additional bucket of custom index data, such as best bets, federal register docs, etc.

Happy to discuss alternatives. This might also just be "good enough" for now.